src/components/ec-alert/ec-alert.vue

```diff
@@ -153,54 +187,87 @@ export default {
     @apply tw-rounded;
     @apply tw-absolute tw-top-8 tw-right-8;
     @apply tw-cursor-pointer;
-    @apply tw-fill-gray-8;
 
     line-height: 0;
 
     &:focus {
       @apply tw-outline-none;
-      @apply tw-bg-gray-8;
     }
 
-    &:hover {
-      @apply tw-bg-gray-8;
+    &:hover,
+    &:focus {
```

Comment:
I came to comment the same than Milos, modifiers can't be appended to pseudoclasses (well, they can with some special syntax in scss, but I really wouldn't use that here.

Could we apply the modifier, then nest the psuedoclass?

Reply:
I've just removed it! It wasn't applying anything and the component was working in the same way